modules/tinymce/src/core/test/ts/browser/FontSelectTest.ts

```diff
@@ -80,6 +80,32 @@ UnitTest.asynctest('browser.tinymce.core.FontSelectTest', function (success, fai
         sAssertSelectBoxDisplayValue(editor, 'Fonts', 'Times')
       ])),
 
+      // https://websemantics.uk/articles/font-size-conversion/
+      Logger.t('TINY-6291: Font size on paragraph with keyword font size is translated to default size', GeneralSteps.sequence([
+        tinyApis.sSetRawContent('<p style="font-family: Times; font-size: medium;">a</p>'),
```

Comment:
Is there a reason you're using `sSetRawContent` here, instead of `sSetContent`? I'm guessing you just copied the line above and that does actually need to not run via the TinyMCE filters, as it'll convert legacy font elements to CSS styles.

If that was the cause, could we change this and the tests below to use `sSetContent` instead?